src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop and run a service locally while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: `sync` or `hybrid` mode. This mode is configured in your Okteto manifest and affects the behavior of [`okteto up`](/reference/cli/#up).
+
+`Sync` is the default mode and changes made to the local service are synchronized with the remote development container deployed in the cluster. Alternatively, `hybrid` mode enables you to run the service in development locally while keeping the rest of the application components in the cloud.
+
+### Deciding which mode to use
+
+There are situations when running a service locally is faster than synchronizing code with a remote development container. For example: when doing frontend-only development or when working in an IDE-heavy context (e.g. Java/Spring) where the IDE may be pre-configured to run tests, start services, and connect debuggers. 
+
+When using Okteto's `hybrid` mode you can maintain your local development experience for the specific service you're working on while still leveraging the benefits of hosting the rest of your application's components in the remote development container.
+
+> Using `hybrid` development mode breaks the replicability between operating systems and increases latency when accessing the rest of the services in the cluster. 
+
+### How hybrid development works
+
+To allow communication to and from the cluster and your local machine an SSH server is deployed in the remote development container. You can expose the local service to the development container using the [`reverse`](/reference/okteto-manifest.mdx/#reverse-string-optional) configuration for the development environment. If you want to forward a service exposed in the cluster to your local machine you can use the [`forward`](/reference/okteto-manifest.mdx/#forward-string-optional) configuration.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/hybrid-development-arch.png").default}
+    alt="Hybrid development architecture"
+    width="850"
+  />
+</p>
+
+When using `hybrid` mode the process running locally is designed to behave similarly as if it were running in `sync` mode. To achieve this the following environment variables are added to the local process running the service in `hybrid` mode. This list is ordered from highest to lowest prevalence:
+- Environment variables declared in the `dev` section of the manifest
+- Environment variables of the remote development container
+- Environment variables of the configmap associated with the dev environment
+- User-defined variables
+- Environment variables declared in the image of the remote development container
```

Comment:
```suggestion
- Environment variables declared in the image of the original container (k8s docs link to the field `image`)
```

Reply:
a link to https://kubernetes.io/docs/concepts/containers/images/ is enough?

Reply:
c254fd0cc7e0892e68bede41146296585d0f0e8e